sceptre/stack.py

```diff
@@ -50,10 +50,11 @@ class Stack(object):
     notifications = ResolvableProperty("notifications")
 
     def __init__(
-        self, name, project_code, template_path, region, iam_role=None,
-        parameters=None, sceptre_user_data=None, hooks=None, s3_details=None,
-        dependencies=None, role_arn=None, protected=False, tags=None,
-        external_name=None, notifications=None, on_failure=None
+            self, name, project_code, template_path, region, iam_role=None,
```

Comment:
It looks like an extra level indent was added here.

Reply:
Strange, I added those because my linter asked me to. I'll revert this.

Reply:
Odd. I'll defer to whatever flake8 says; but, since flake8 decided the old version is fine, I'm surprised your linter asked to indent another level.

Reply:
Seems like flake8 finds both acceptable. I used pylint, they may defer on some details I guess.

Reply:
I suppose the parameters are within parenthesis so the linter might not care? In any case I would agree with @b-t-g that this should have the additional level of indentation removed.